### PR TITLE
Adding late fees to CS registration test page

### PR DESCRIPTION
**Pull request by paulshimmen**

### CODE REVIEW

**Review by manishagarwalEviden — APPROVED** [review]

File: src/EPR.Payment.Facade/wwwroot/registrationFees.html [hunk 1]
```diff
@@ -83,7 +91,11 @@ <h2>Members</h2>
                     <input type="number" id="onlineMarketplaceSubsidiaryCount3">
                     <div class="checkbox-container">
                         <input type="checkbox" id="isOnlineMarketplaceMember3">
-                        <label for="isOnlineMarketplaceMember3">Is Member Online Marketplace</label>
+                        <label for="isOnlineMarketplaceMember3">Is Member Online Marketeeplace</label>
```
Comment (davidjayakumarEviden) [inline src/EPR.Payment.Facade/wwwroot/registrationFees.html:94]: Typo Marketeeplace

File: src/EPR.Payment.Facade/wwwroot/js/registrationFees.js [hunk 2]
```diff
@@ -165,23 +167,32 @@ function displayFees(data) {
...
+
         return `
             <h3>Member ID: ${member.memberId}</h3>
             <table>
                 <tbody>
                     <tr class="reg-info">
                         <td>Member Registration Fee</td>
                         <td>£${formatCurrency(member.memberRegistrationFee)}</td>
-                    </tr>
+                    </tr>
```
Comment (davidjayakumarEviden) [inline src/EPR.Payment.Facade/wwwroot/js/registrationFees.js:185]: Empty space can be removed

File: src/EPR.Payment.Facade/wwwroot/js/registrationFees.js [hunk 3]
```diff
@@ -165,23 +167,32 @@ function displayFees(data) {
...
                         <td>Member Online Marketplace Fee</td>
                         <td>£${formatCurrency(member.memberOnlineMarketPlaceFee)}</td>
                     </tr>
+                    ${lateFeeRow}
                     <tr class="reg-info">
                         <td>Subsidiaries Fee</td>
                         <td>£${formatCurrency(member.subsidiariesFee)}</td>
-                    </tr>
-                    ${breakdownRows}
+                    </tr>
```
Comment (davidjayakumarEviden) [inline src/EPR.Payment.Facade/wwwroot/js/registrationFees.js:194]: Remove empty spaces also in the line below

File: src/EPR.Payment.Facade/wwwroot/registrationFees.html [hunk 4]
```diff
@@ -83,7 +91,11 @@ <h2>Members</h2>
                     <input type="number" id="onlineMarketplaceSubsidiaryCount3">
                     <div class="checkbox-container">
                         <input type="checkbox" id="isOnlineMarketplaceMember3">
-                        <label for="isOnlineMarketplaceMember3">Is Member Online Marketplace</label>
+                        <label for="isOnlineMarketplaceMember3">Is Member Online Marketeplace</label>
```
Comment (davidjayakumarEviden) [inline src/EPR.Payment.Facade/wwwroot/registrationFees.html:94]: Marketeplace, still one more e to be removed

**Review by davidjayakumarEviden — APPROVED** [review]
Looks good

**Pull request merged**